### ISSUE TITLE
Devolver sólo el dato de interés y no el json completo

### ISSUE THREAD

**Issue by jgsogo** [jgsogo/apicultur-python#4]
Una funcionalidad deseable por el cliente de la librería (especialmente si no quiere complicarse la vida)  es que las llamadas a `apicultur` devuelvan directamente el dato que le interesa en vez del json completo con la respuesta de la API.

Así, este código a:

```python
>>> data = apicultur.lematiza2(word='meses')
>>> print(data)
{  
    'palabra':'meses',
    'lemas':[  
        {  
            'lema':'mes',
            'categoria':'NCMP000'
        },
        {  
            'lema':'mesar',
            'categoria':'VMSP2S0'
        }
    ]
}
```

podría pasar a funcionar de la siguiente forma:

```python
>>> data = apicultur.lematiza2(word='meses', category='N')
>>> print(data)
mes
```

o bien

```python
>>> data = apicultur.lematiza2(word='meses', category='N')
>>> print(data)
('mes', 'NCMP000')
```

o bien algún tipo de estructura que diseñemos para almacenar el lema y su categoría (eventualmente más información) y que permita acceder a los datos de forma conveniente.

En el parámetro `category` (o el correspondiente en cada caso) habría que pasarle la información necesaria para desambiguar entre las posibles respuestas.

¿Nos gusta? ¿Nos gusta sólo para el ApiculturSafe propuesto en #2? ¿Nos gusta, pero no nos convence?

**Comment by ebaste** [comment]
Yo creo que no es operativo. Obtener la información de un JSON en más complejo al principio, pero en cuanto encuentras la lógica es fácil, y el saberlo hacer te da mucho potencial.

Creo que sería más práctico hacer un pequeño tutorial con ejemplos

**Comment by jgsogo** [comment]
Estoy de acuerdo. De momento no cierro el issue para que la gente lo vea, lo lea y opine si quiere, pero sí que le voy a añdir la etiqueta `wontfix`.

----
Sobre `json` hay que hacer algo cuanto antes (tutorial o taller). En el repositorio que le he preparado a Laura los datos de partida los he puesto en ese formato ([ver](https://github.com/Lingwars/billetes-tren/blob/master/data/mensajes.json)) y hay que justificar por qué utilizar algo tan complicado (creo issue [aquí](https://github.com/Lingwars/blog/issues/10)).
